terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1112,7 +1112,7 @@ def remove_inconsequential_statements(statements: List[JSON]) -> List[JSON]:
                     mounts:
                     - ["/dev/nvme1n1", "/mnt/gitlab", "ext4", ""]
                     rancher:
-                    ssh_authorized_keys: {other_public_keys if config.deployment_stage == 'dev' else []}
```

Comment:
not sure why this line was changed. the commit title is also off.

Reply:
Sorry, some sort of git snafu. I wanted this change to be squashed in, since it fixes the line length. Anyway I've corrected the history as you suggested.